lib/Doctrine/ORM/Mapping/InverseJoinColumn.php

```diff
@@ -35,14 +35,21 @@ final class InverseJoinColumn implements Annotation
      */
     public $fieldName;
 
+    /** @var array<string,mixed> */
```

Comment:
If the list is the same as above, we could create a `psalm-type` (we have some in the codebase already), and reuse it where it makes sense to do so.

Reply:
Not possible, see above.